manifests/extension_pack.pp

```diff
@@ -26,11 +26,12 @@
 #  The path to the VBoxManage command, defaults to '/usr/bin/VBoxManage'.
 #
 class vbox::extension_pack(
-  $ensure     = 'present',
-  $source     = $vbox::params::extension_pack_url,
-  $pack       = $vbox::params::extension_pack,
-  $directory  = $vbox::params::extension_pack_dir,
-  $vboxmanage = $vbox::params::vboxmanage,
+  $ensure                 = 'present',
+  $source                 = $vbox::params::extension_pack_url,
+  $pack                   = $vbox::params::extension_pack,
+  $directory              = $vbox::params::extension_pack_dir,
+  $vboxmanage             = $vbox::params::vboxmanage,
+  $license_agreement_hash,
```

Comment:
I think puppet isn't happy with args without defaults after args with them, or did I dream that? I think a default of `undef` would be fine. Could we support with and without?